### PR TITLE
Merge 1.x into Master

### LINKED ISSUE #83: Create photo carousel shortcode embed
The goal of this issue is to create a way for users to create and use a list of photo IDs to render a photo carousel inside their content. This will utilize the [Swiper.js](https://swiperjs.com/) touch-based slider. 

Proposed example usage: `[wsuwp_photo_carousel ids="123,456,789"]`

### PR DESCRIPTION
**Pull request by nicdford**
- Adds support for [wsuwp_photo_carousel] shortcodce

Closes #83

### CODE REVIEW

File: displays/photo-carousel.php [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * @var $ids array IDs for the images to be displayed in the carousel
+ * @var $name string Returns the name of the current instance from the shortcode params or the current page id (limits usage to one per page)
+ */
+?>
+
+<!-- Swiper -->
+<style>
```
Comment (danbleile) [inline displays/photo-carousel.php:10]: Do you need the style here? Is this included in swiper.min.css? If you do that's fine - just wanted to make sure it wasn't duplicated.

File: displays/photo-carousel.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+		padding: 1em .8em 1em .5em;
+	}
+
+	/* Pagination */
+	.swiper-container .swiper-pagination-bullet-active {
+		background: #ca1237;
+	}
+</style>
+
+<div id="swiper" class="swiper-container swiper_<?php echo $name; ?>">
```
Comment (danbleile) [inline displays/photo-carousel.php:84]: The output will need to be sanitized here - esc_html(), esc_attr(), esc_url(), etc....

It's good practice to sanitize output even if it's coming from WP (as is the case with $image_url).

File: displays/photo-carousel.php [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+	<div class="swiper-button-prev"></div>
+
+	<!-- Add Pagination -->
+	<div class="swiper-pagination"></div>
+</div>
+<!-- Swiper End -->
+
+
+
+<script>
```
Comment (danbleile) [inline displays/photo-carousel.php:106]: Check out wp_localize_script and wp_add_inline_script if you haven't already (don't expect any changes here just thoughts for the future) https://developer.wordpress.org/reference/functions/wp_localize_script/
https://developer.wordpress.org/reference/functions/wp_add_inline_script/

File: displays/photo-carousel.php (hunk 1 above)
Comment (nicdford) [reply displays/photo-carousel.php:10]: These styles are a combination of being specific to a "carousel", but also include some "WSU branded" overrides. Also, these styles only help the "tile" view we're going for, so I thought I made sense to put them here and then we can know to move them into the block once we get there.

File: displays/photo-carousel.php (hunk 2 above)
Comment (nicdford) [reply displays/photo-carousel.php]: Got it!

File: displays/photo-carousel.php (hunk 3 above)
Comment (nicdford) [reply displays/photo-carousel.php:104]: Sounds good!

**Pull request merged**